common/etc/functions

```diff
@@ -187,6 +207,10 @@ wwreqroot() {
     fi
 }
 
+wwishost() {
```

Comment:
@jtfrey It appears the function wwishost() was half completed, and isn't actually used anywhere. Can it be removed?